### DISCUSSION TITLE
SSL Error when attempting upload to du-itc.itunes.apple.com due on AWS machine

### DISCUSSION THREAD

**Discussion by addbrick** [fastlane/fastlane#18188] [General]
I am looking for anyone else who has experienced this, has an explanation of what is happening or a better solution than including an older cert.

**TLDR:**  Uploading screenshots to ` du-itc.itunes.apple.com` broke on AWS machines but worked locally. Fixed by including old 2014 `ca-bundle.crt` from the ruby gem [Certified](https://github.com/stevegraham/certified).

My production system, running on AWS, since last week (Feb 4th) started to run into this error `SSL_connect returned=1 errno=0 state=error: certificate verify failed (Faraday::SSLError)` when attempting to upload the review screenshot for In-App Purchases. Creating and updating all other metadata worked with no issue. I was able to upload screenshot from my local machine with the same production code. After digging into the code I found the image upload domain `du-itc.itunes.apple.com` and attempted to curl it. Got the same error 
```bash
-> curl --location --request POST 'https://du-itc.itunes.apple.com/upload/image' -I
curl: (60) SSL certificate problem: unable to get local issuer certificate

More details here: https://curl.haxx.se/docs/sslcerts.html
curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```

After attempting to install different certs and ensuring everything was up to date, I stumbled upon this ruby gem [Certified](https://github.com/stevegraham/certified). It contains an older `ca-bundle.crt` from 2014. I included it in the test script on the AWS machine and it worked. So I am guessing either the Apple domain ` du-itc.itunes.apple.com` is misconfigured or I just couldn't find the right cert.

Has anyone else experienced this?

My main question is, what happened at the beginning of February to cause these connections issues? I double checked my system and nothing was updated since mid January. I did see Apple recently [updated their Apple Worldwide Developer Relations Intermediate Certificate](https://developer.apple.com/support/wwdr-intermediate-certificate/?utm_campaign=iOS%2BDev%2BWeekly&utm_medium=email&utm_source=iOS%2BDev%2BWeekly%2BIssue%2B492). This did cause issues on another CI system of mine, but was solved by updating the certs on that Mac machine.

Second question does anyone have a better solution than using this old 2014 cert for the image upload?
